### PR TITLE
Rename class OlapMeta to KVStore

### LINKED ISSUE #74: Rename OlapMeta to KVStore
https://github.com/StarRocks/starrocks/blob/dbb993b476f3964ea8d06811b36716e6b5b3c0ff/be/src/storage/olap_meta.h#L45
The class `OlapMeta` is a thin wrapper on RocksDB to provides key-value storage interfaces, the `OlapMeta` is somehow inaccurate, suggest renaming it to `KVStore`.

### PR DESCRIPTION
**Pull request by BanBuDu0**
Fix #74

### CODE REVIEW

**Comment by BanBuDu0** [comment]
I run 
`cd build-support/
sh clang-format.sh`
this in my mac Os catalina 10.15.7 clang-format-12.0.1
and still have starrocks_clang-format error

**Comment by sduzh** [comment]
> I run
> `cd build-support/`
> `sh clang-format.sh`
> in my mac Os catalina 10.15.7 clang-format-12.0.1
> and still have starrocks_clang-format error

Good Job!

The clang-format we are using is 10.0.1，can you install the same version on your local computer? We'll try to provide the clang-format along with the source code or docker image in the near future.

File: be/src/storage/kv_store.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This file is made available under Elastic License 2.0.
 // This file is based on code available under the Apache license here:
-//   https://github.com/apache/incubator-doris/blob/master/be/src/olap/olap_meta.cpp
+//   https://github.com/apache/incubator-doris/blob/master/be/src/olap/kv_store.cpp
```
Comment (sduzh) [inline be/src/storage/kv_store.cpp:3]: This should not be modified.

File: be/src/storage/kv_store.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // This file is made available under Elastic License 2.0.
 // This file is based on code available under the Apache license here:
-//   https://github.com/apache/incubator-doris/blob/master/be/src/olap/olap_meta.h
+//   https://github.com/apache/incubator-doris/blob/master/be/src/olap/kv_store.h
```
Comment (sduzh) [inline be/src/storage/kv_store.h:3]: This should not be modified

File: be/test/storage/kv_store_test.cpp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // This file is made available under Elastic License 2.0.
 // This file is based on code available under the Apache license here:
-//   https://github.com/apache/incubator-doris/blob/master/be/test/olap/olap_meta_test.cpp
+//   https://github.com/apache/incubator-doris/blob/master/be/test/olap/kv_store_test.cpp
```
Comment (sduzh) [inline be/test/storage/kv_store_test.cpp:3]: This should not be modified.

**Review by sduzh — APPROVED** [review]

**Comment by sduzh** [comment]
LGTM, PTAL @imay

**Comment by BanBuDu0** [comment]
why starrocks_be_unittest error after I change some code annotation  /(ㄒoㄒ)/~~

**Comment by sduzh** [comment]
> why starrocks_be_unittest error after I change some code annotation /(ㄒoㄒ)/~~

The tests are not quite stable now, you can rerun it by typing `run starrocks_be_unittest`

**Pull request merged**